edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/ConnectionHandler.cs

```diff
@@ -94,6 +99,20 @@ public async Task<bool> HandleAsync(MqttPublishInfo publishInfo)
             return false;
         }
 
+        public async Task CloseConnectionAsync(IIdentity identity)
+        {
+            var identityFound = false;
+            using (await this.guard.LockAsync())
+            {
+                identityFound = this.knownConnections.TryGetValue(identity, out var _);
+            }
+
+            if (identityFound)
+            {
+                await this.connector.SendAsync(TopicDisconnect, Encoding.UTF8.GetBytes($"\"{identity.Id}\""));
+            }
```

Comment:
If the identity isn't found should we log error?

Reply:
I am not sure if that is an error. Can a device be gone by the time edgeHub realizes that something is wrong a calls Close()? I can add an 'Info'.